File: dsa/3-arrays/challenges/06-two-sum/two-sum.py
def two_sum(arr, target):
    result = []

    for i in range(len(arr)):
        for j in range(i + 1, len(arr)):
            if arr[i] + arr[j] == target:
                result.append([i, j])

    print(result) if len(result) > 0 else print("not a match")


two_sum([2, 7, 11, 15, 5, 4], 9)

File: dsa/3-arrays/challenges/06-two-sum/twoSum.js
function twoSum(arr, target) {
    let result = []

    for (let i = 0; i < arr.length; i++) {
        for (let j = i + 1; j < arr.length; j++) {
            if (arr[i] + arr[j] === target) {
                result.push([i, j])
            }
        }
    }
    result.length > 0 ? console.log(result) : console.log('not a match')
}

twoSum([2, 7, 11, 15, 5, 4], 200)
